client/layout/masterbar/checkout.tsx

```diff
@@ -52,6 +53,12 @@ const CheckoutMasterbar = ( {
 			return 'akismet';
 		}
 
+		// Temporary workaround to test the branding for Passport checkout page.
+		// TODO: Remove this once we have a better way to identify Passport checkout pages.
+		if ( window.location.href.includes( 'passport' ) ) {
```

Comment:
Should this updated to `/checkout/passport` and remove TODO: comment?
```suggestion
		if ( window.location.pathname.startsWith( '/checkout/passport' ) ) {
```